Index: src/code/Dashboard_Admin.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package code;\r\n\r\nimport javafx.application.Application;\r\nimport javafx.fxml.FXMLLoader;\r\nimport javafx.scene.Parent;\r\nimport javafx.scene.Scene;\r\nimport javafx.scene.image.Image;\r\nimport javafx.stage.Stage;\r\n\r\nimport java.util.Objects;\r\n\r\npublic class Dashboard_Admin extends Application{\r\n\r\n\r\n        @Override\r\n        public void start(Stage stage) throws Exception {\r\n            Parent root = FXMLLoader.load(Objects.requireNonNull(getClass().getResource(\"../resource/code.admin/dashboard_admin.fxml\")));\r\n            stage.setTitle(\"All IN ONE STORE - Admin Dashboard\");\r\n            stage.getIcons().add(new Image(\"src/img/icon.png\"));\r\n            stage.setScene(new Scene(root,1500,820));\r\n            stage.show();\r\n        }\r\n\r\n        public static void main(String[] args) { launch(args);}\r\n\r\n\r\n\r\n    }\r\n\r\n\r\n\r\n\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/code/Dashboard_Admin.java b/src/code/Dashboard_Admin.java
--- a/src/code/Dashboard_Admin.java	
+++ b/src/code/Dashboard_Admin.java	
@@ -14,7 +14,7 @@
 
         @Override
         public void start(Stage stage) throws Exception {
-            Parent root = FXMLLoader.load(Objects.requireNonNull(getClass().getResource("../resource/code.admin/dashboard_admin.fxml")));
+            Parent root = FXMLLoader.load(Objects.requireNonNull(getClass().getResource("../resource/dashboard_admin.fxml")));
             stage.setTitle("All IN ONE STORE - Admin Dashboard");
             stage.getIcons().add(new Image("src/img/icon.png"));
             stage.setScene(new Scene(root,1500,820));
